exchanges/exchange.go

```diff
@@ -31,41 +31,42 @@ const (
 )
 
 // FeeType custom type for calculating fees based on method
-type FeeType string
-
-// InternationalBankTransactionType custom type for calculating fees based on fiat transaction types
-type InternationalBankTransactionType string
+type FeeType int
 
 // Const declarations for fee types
 const (
-	BankFee                        FeeType = "bankFee"
-	InternationalBankDepositFee    FeeType = "internationalBankDepositFee"
-	InternationalBankWithdrawalFee FeeType = "internationalBankWithdrawalFee"
-	CryptocurrencyTradeFee         FeeType = "cryptocurrencyTradeFee"
-	CyptocurrencyDepositFee        FeeType = "cyptocurrencyDepositFee"
-	CryptocurrencyWithdrawalFee    FeeType = "cryptocurrencyWithdrawalFee"
+	BankFee                        FeeType = 0
```

Comment:
This can be simplified to iota and the values below will automatically be incremented